topic04-git-introduction/book-git-intro/03.Basic-git-commands.md

```diff
@@ -0,0 +1,22 @@
+# Basic Git Commands
+
+- `git clone <ssh address>`
+  - This creates a new folder inside the folder you are in using the same name as the remote repository and containing any code stored in the remote repository.
+
```

Comment:
would add a newline here (needs a double space to render as a newline in markdown)